gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/TransactionalService.java

```diff
@@ -15,12 +15,11 @@
  */
 package io.gravitee.rest.api.service.impl;
 
-import io.gravitee.rest.api.service.exceptions.AbstractManagementException;
 import org.springframework.transaction.annotation.Transactional;
 
 /**
  * @author Azize ELAMRANI (azize.elamrani at graviteesource.com)
  * @author GraviteeSource Team
  */
-@Transactional(value = "graviteeTransactionManager", noRollbackFor = AbstractManagementException.class)
```

Comment:
This PR is not about that and shouldn't involve such a change, especially because you are also removing the transactional configuration.